app/frontend/stores/permit-application-store.ts

```diff
@@ -184,8 +184,8 @@ export const PermitApplicationStoreModel = types
     },
   }))
   .actions((self) => ({
-    createPermitApplication: flow(function* (formData: TCreatePermitApplicationFormData) {
-      const { ok, data: response } = yield self.environment.api.createPermitApplication(formData)
+    createPermitApplication: flow(function* (formData: TCreatePermitApplicationFormData, sandboxId: string = null) {
+      const { ok, data: response } = yield self.environment.api.createPermitApplication(formData, sandboxId)
```

Comment:
does this work? it looks like `createPermitApplication` takes in a single `params` argument so I would expect `sandboxId` would need to be merged with `formData` in order for this to get passed correctly?

Reply:
this has been removed in a later branch. sandBox ID should always come from the header

Reply:
OK - in the future, try to keep changes/fixes in the same PR if the PR hasn't been merged into dev yet - this makes review a lot easier